_includes/mobile/video-params.md

```diff
@@ -28,7 +28,7 @@ The [OpenRTB v2.6-202303](https://github.com/InteractiveAdvertisingBureau/AdCOM/
 - `4` or `NoContent` or `Standalone` : Video ads that are played without streaming video content. This can be in placements like slideshows, native feeds, in-content or sticky/floating.
 
 Notes:
-- `PrebidAdUnit`, `VideoInterstitialAdUnit` and `RewardedVideoAdUnit` will default to plcmnt=3 if no placement value is supplied.
+- `PrebidAdUnit` and `RewardedVideoAdUnit` will default to plcmnt=3 if no placement value is supplied.
```

Comment:
Is this `plcmt` or `placement` like the one above? There's a difference. (And if plcmt, note the typo here.)